hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/src/main/java/org/apache/hadoop/yarn/server/federation/store/records/SubClusterState.java

```diff
@@ -55,6 +55,10 @@ public boolean isUnusable() {
     return (this != SC_RUNNING && this != SC_NEW);
   }
 
+  public boolean isUsable() {
+    return !isUnusable();
```

Comment:
Can we just use this directly?
Let's just have one of them either isUsable or isUnusable.

Reply:
Thank you very much for helping to review the code, I will modify the code.